layouts/partials/footer_js.html

```diff
@@ -31,15 +29,5 @@
       type: "newsItems"
     });
   {{ end }}
-  {{ if ne .Page.Params.show_featured_story false }}
```

Comment:
Why are we removing this? This functionality is how we auto trigger newsroom components